packages/client/bin/cli.ts

```diff
@@ -432,7 +433,7 @@ async function startClient(config: Config, customGenesisState?: GenesisState) {
   if (customGenesisState !== undefined) {
     const validateConsensus = config.chainCommon.consensusAlgorithm() === ConsensusAlgorithm.Clique
     blockchain = await Blockchain.create({
-      db: dbs.chainDB,
+      db: new LevelDB(dbs.chainDB) as any,
```

Comment:
`as any` necessary here?

Reply:
It isn't here.  It is in certain other places since we re-use the `db` from the client across lower level modules and they have their own particular key/value typing that is a subset (or different in the `Ethash` case) than what the `client` uses, though the underlying database can support either.